stable/jenkins/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version 1.5.7 is auto generated based on git commits. Those include a reference to the git commit to be able to get more details.
 
+## 1.10.2 Support multiple agents in JCasC
```

Comment:
I think this is worth a minor version increment.

Reply:
Will do.